sfge/src/main/java/com/salesforce/Main.java

```diff
@@ -129,6 +130,11 @@ private int execute(String... args) {
             LOGGER.error("Error while loading graph", ex);
             System.err.println(formatError(ex));
             return INTERNAL_ERROR;
+        } catch (UnexpectedException ex) {
+            LOGGER.error("Unexpected exception while loading graph", ex);
```

Comment:
New catch-all for Unexpected exceptions so that the error message is less cryptic.